microsoftgraph/microsoft-graph-explorer-v4
# Chore: Dependabot upgrades for February

## Overview

Updates various packages from dependabot PR's
Fixes #2998 

### Demo

Optional. Screenshots, `curl` examples, etc.

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* Check out this branch
* Load GE
* Test all major functionality to ensure no breaking changes
